Depth-Anything-V2-list3/depth_anything_v2/util/blocks.py

```diff
@@ -46,9 +49,8 @@ def __init__(self, features, activation, bn):
         
         self.conv2 = nn.Conv2d(features, features, kernel_size=3, stride=1, padding=1, bias=True, groups=self.groups)
 
-        if self.bn == True:
-            self.bn1 = nn.BatchNorm2d(features)
-            self.bn2 = nn.BatchNorm2d(features)
+        self.bn1 = nn.BatchNorm2d(features) if bn else nn.Identity()
+        self.bn2 = nn.BatchNorm2d(features) if bn else nn.Identity()
```

Comment:
Why the change?

Reply:
The TorchScript exporter does not allow conditional initialization. Either initialize or don't. I initialized it to `nn.Identity()`, and not `None`, to prevent type mismatch errors.